formatters/esformatter.py

```diff
@@ -0,0 +1,188 @@
+#!/bin/env/python3
+import sqlite3
+import json
+import sys
+
+from elasticsearch import Elasticsearch, helpers
+
+path_to_db = sys.argv[1]
+
+es = Elasticsearch()
+connect = sqlite3.connect(path_to_db)
+
+def push_messages():
```

Comment:
This should subclass the `BaseFormatter` class, if it's going to be under `formatters/`.